src/test/common/page/enter-children-dob.js

```diff
@@ -9,6 +9,12 @@ const PAGE_TITLES = {
   cy: 'GOV.UK - Urna condimentum mattis?'
 }
 
+const getChildDobDayInputIdForIndex = (index) => `childDob-${index}-day`
+const getChildDobMonthInputIdForIndex = (index) => `childDob-${index}-month`
+const getChildDobYearInputIdForIndex = (index) => `childDob-${index}-year`
+const getChildDobNameInputIdForIndex = (index) => `childDob-name-${index}`
+const getRemoveChildButtonIdForIndex = (index) => `remove-child-${index}`
```

Comment:
Minor suggestion (that shouldn't stop this being merged, and is a matter of opinion): As these are only used within this file the names could be a bit shorter, e.g. `getChildDobDayInputIdForIndex` -> `getDayInputIdForIndex`

Reply:
Yep, good suggestion, done.